convert2rhel/checks.py

```diff
@@ -342,7 +342,7 @@ def check_custom_repos_are_valid():
         return
 
     output, ret_code = call_yum_cmd(
-        command="makecache",
+        command="repolist",
         args=["-v", "--setopt=*.skip_if_unavailable=False"],
         print_output=False,
     )
```

Comment:
I always got confused with this function. @bocekm should this check be related to the repos we received through `--enablerepo` in convert2rhel, or, what type of repo? Reading the function docstring it seems to be that we should check the ones that are provided by the user, not the ones that we have in the system.

Reply:
This is indeed related to the RHEL repoid(s) provided by the user through the `--enablerepo` option. We check if they are accessible because if the provided repoids point to some custom repo location, let's say to the user's FTP server where they mirror the RHEL repos from Red Hat CDN, we want to make sure they have set it up correctly before proceeding. Or to spot even things like typos in the .repo file they've crafted.

Reply:
Ah, I see. 

Yeah, having the enable repos being enabled under the hood is quite confusing sometimes. Maybe in the future we could make it more explicit and bring that up?